sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -246,7 +250,7 @@ namespace Azure { namespace Core {
     {
       if (IsCancelled())
       {
-        throw OperationCancelledException("Request was cancelled by context.");
+        throw OperationCancelledException("Request was cancelled by context."); // LCOV_EXCL_LINE
```

Comment:
Same here. Why are we unable to test this?

Reply:
It is not that we don't test the exception being thrown, it is construction of a string from a string literal.